### ISSUE TITLE
bug: variables named "get" produce kw_get errors

### ISSUE THREAD

**Issue by terite** [quick-lint/quick-lint-js#55]
When linting the following file

```javascript
let get;
```

I expect quick-lint-js to produce no diagnostics.

But instead it tells me:
```
get.js:1:1: error: let with no bindings
get.js:1:4: error: missing semicolon after expression
<...>/src/./quick-lint-js/parse.h:210: fatal: token not implemented in parse_and_visit_statement: kw_get on line 1 column 5
```

This issue feels distinct from #28 because it produces two erroneous diagnostics before crashing.

**Comment by strager** [comment]
As of commit b9e4851, the code is parsing without a crash and with no diagnostics.
